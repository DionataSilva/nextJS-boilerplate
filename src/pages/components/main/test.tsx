import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  it('should render te heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /meu primeiro teste!/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
