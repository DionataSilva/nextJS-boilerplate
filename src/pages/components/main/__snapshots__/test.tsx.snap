// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render te heading 1`] = `
<main>
  <h1>
    meu primeiro teste!
  </h1>
</main>
`;
